components/html-block/html-block.js

```diff
@@ -245,8 +245,10 @@ class HtmlBlock extends LitElement {
 			if (fragment) {
 
 				let temp = document.createElement('div');
+				temp.style.display = 'none';
 				temp.appendChild(fragment);
 
+				this._renderContainer.appendChild(temp);
```

Comment:
I have no idea if this is the best solution, so if there's something better I can do here, I'm happy to consider alternatives. My immediate thoughts were that we could check the existing document font size directly or something of the sort, but that becomes problematic because MathJax acts iteratively on the document it's provided... In other words, the font size it computes will still depend on the surrounding HTML, so I don't think we can reasonably "fake" it.

Reply:
Note that I _do_ believe this should be reasonably safe. Two lines below we overwrite the render container entirely, so this temp element will get wiped out as soon as processing finishes.